about.html

```diff
@@ -0,0 +1,142 @@
+
+                    <ul class="nav-ul">
+                        <li><a href="index.html">Home</a></li>
+                        <li><a href="about.html">About</a></li>
+                        <li>Program</li>
+                        <li>Join</li>
+                        <li>Sponsor</li>
+                        <li>News</li>
+                        <li><h3 class="winners">Winners Chapel</h3></li>
+                    </ul>
+                </nav>
+            </div>  
+        </div>
+        
+    </header>
+        <article class="intro">
+          <div class="menu-nav">
+                <div class="hamburger"></div>
+                <div class="hamburger"></div>
+                <div class="hamburger"></div>
+          </div>
+
+          <div class="mobileMenu"> 
+            <div class="m1"><h2 class="m2">Winners Chapel</h2></div>
+            <span class="closeMenu">&times;</span>
+              <ul class="container">
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="index.html">Home</a></li>
+                    <li>Program</li>
+                    <li>Join</li>
+                    <li>News</li>
+             </ul>
+            </div>
+
+        </article>
+    <main class="pg1">
+       <a href=""><img src="./images/about-bg.png" alt="" class="bckg1"></a>
+            <h2 class="theme">
+                "Youth For Christ"
+             </h2>
+             <h1 class="ch-name">Winners Chapel Youth Conference 2022.</h1>
+            <div class="moment">
+                 <p class="prg">the conference is to bring together all youth in all church in the nation under one canmopy to worship God.</p>
+            </div>
+            <div class="contact">
+                <p>please contact us per email for any further questions about winners chapel youth conference! </p>
+                <a  href="">winnerconf@gmail.com</a>
+            </div>
+        </main>
+    
+        <div>
+           <div class="logo1">
+
+                <h3> winners chapel youth conference 2022 logo</h3>
+                <p> winners chapel youth conference 2022 logo ws decided by the youth president bhimself alongside some executives</p>
+            
+            <div class="logo-p">
+                <a href=""><img src="./images/download.jpg" alt="" class="logo-ab"></a>
+            </div>
+        </div>
+        <hr class="line"> 
+    </div>
+<section>
+        <h3 class="abh2">See The Past National Youth Conference</h3>
+       <hr class="hr-f"> 
+        <p class="abp">Take a look at the past two conference which took place in abuja, nigeria</p>
+  <section class="di">    
+<div class="card">
+    <div class="image">
+        <div class="bckg">
+        </div>   
+    </div>
+    <div class="text" >
+    <p class="da">2021 </p>
+<p> youth national conference in lagos</p>
+</div>
+</div>
```

Comment:
- [ ] Please make sure to give a proper alignment for all the elements rendered on the screen. Please git rid of that offset

![image](https://user-images.githubusercontent.com/52635378/164945785-ba174fe7-32f8-4504-beaa-1f09dacc61c7.png)

Reply:
please.. on my screen its displaying okay

Reply:
done